app.js

```diff
@@ -1,18 +1,29 @@
 const bodyParser = require("body-parser");
 const express = require("express");
+const session = require("express-session");
 const cors = require("cors")
 const swaggerJsdoc = require('swagger-jsdoc');
 const swaggerUi = require('swagger-ui-express');
 require("dotenv").config();
-require("./auth/passport");
+
 
 require('./models/user');
 
-const middlewares = require('./middlewares');
 const api = require ('./api');
+
+function isLoggedIn(req, res, next) {
```

Comment:
this is not correct to use a " function " statement
The recommendation is to use const, so 

const isLoggedIn = () => {}